samples/subsys/usb/hid-mouse/src/main.c

```diff
@@ -92,75 +149,178 @@ static const u8_t hid_report_desc[] = {
 		/* END_COLLECTION				C0    */
 };
 
-static int get_report_cb(struct usb_setup_packet *setup, s32_t *len,
```

Comment:
Based on our discussion on Slack, this is a bug. I prefer move this fix to another PR.